src/databricks/labs/remorph/install.py

```diff
@@ -21,80 +20,156 @@
 
     def run(self):
-        logger.info(f"Installing Remorph v{PRODUCT_INFO.version()}")
-        config = self.configure()
+        logger.info(f"Installing Remorph v{self._product_info.version()}")
+        configs = self.configure()
         workspace_installation = WorkspaceInstallation(
-            config,
+            configs,
             self._installation,
             self._ws,
             self._prompts,
             verify_timeout=timedelta(minutes=2),
+            product_info=self._product_info,
         )
-        workspace_installation.run()
-
-    def configure(self) -> MorphConfig:
         try:
-            return self._installation.load(MorphConfig)
-        except NotFound as err:
-            logger.debug(f"Cannot find previous installation: {err}")
-        logger.info("Please answer a couple of questions to configure Remorph")
-
-        # default params
-        catalog_name = "transpiler_test"
-        schema_name = "convertor_test"
-        ws_config = None
+            workspace_installation.run()
+        except ManyError as err:
+            if len(err.errs) == 1:
+                raise err.errs[0] from None
+            raise err
+        return configs
 
-        source_prompt = self._prompts.choice("Select the source", list(SQLGLOT_DIALECTS.keys()))
-        source = source_prompt.lower()
+    def configure(self) -> RemorphConfigs:
+        """
+        Returns the MorphConfig If it exists on the Databricks Workspace,
+         else prompts for the new Installation
+        :return:
+        """
+        morph_config = None
+        reconcile_config = None
 
-        skip_validation = self._prompts.confirm("Do you want to Skip Validation")
+        try:
+            morph_config = self._installation.load(MorphConfig)
+        except NotFound as err:
+            logger.debug(f"Cannot find previous `transpile` installation: {err}")
+        # except (PermissionDenied, SerdeError, ValueError, AttributeError): logger.warning(f"Existing installation
+        # at {self._installation.install_folder()} is corrupted. Skipping...")
 
-        if not skip_validation:
-            ws_config = self._configure_runtime()
-            catalog_name = self._prompts.question("Enter catalog_name")
-            try:
-                self._catalog_setup.get(catalog_name)
-            except NotFound:
-                self.setup_catalog(catalog_name)
+        try:
+            reconcile_config = self._installation.load(ReconcileConfig)
+        except NotFound as err:
+            logger.debug(f"Cannot find previous `reconcile` installation: {err}")
+        # except (PermissionDenied, SerdeError, ValueError, AttributeError): logger.warning(f"Existing installation
+        # at {self._installation.install_folder()} is corrupted. Skipping...")
 
-            schema_name = self._prompts.question("Enter schema_name")
+        if morph_config or reconcile_config:
+            return RemorphConfigs(morph_config, reconcile_config)
```

Comment:
Why do we need two config files? This will turn into maintenance nightmare

Reply:
As we have two separate modules for specific functionalities:
* `transpile` -  used for converting scripts 
* `reconcile` - used for Data reconciliation between Source and Target.

We are storing the config specific to each module in a separate file: `config` and `reconcile`